speed-transition.lua

```diff
@@ -91,6 +100,19 @@ function toggle_sub_visibility()
    toggle2 = not toggle2
 end
 
+function toggle_mode()
+   skipmode = not skipmode
+   if enable then
+      toggle()
+      toggle()
```

Comment:
checking for `not enable` and dropping one `toggle()` should be more intuitive

Reply:
This is so if the mode is switched it will do it immediately. Otherwise it has to wait for the next sub event to change.